erpnext/stock/report/stock_balance/test_stock_balance.py

```diff
@@ -53,9 +52,11 @@ def assertInvariants(self, rows):
 						ORDER BY timestamp(posting_date, posting_time) desc, creation desc)
 						AS rn
 					FROM `tabStock Ledger Entry`
-					where is_cancelled=0
+					where posting_date >= %s and posting_date <= %s and is_cancelled=0
```

Comment:
Never modify test code to them like this! It is encoding a known supported business use case. 😅